lib/conv/msgpack_conv.ml

```diff
@@ -11,7 +11,7 @@ include Meta_conv.Coder.Make(struct
     let tuple =
       Encode.array
 
-    let variant tag _ = function
+    let variant _ tag = function
```

Comment:
first argument is type name, and second argument is tag name.
https://bitbucket.org/camlspotter/meta_conv/diff/json/json_conv.ml?diff2=b66a3e869fee&at=default